pom.xml

```diff
@@ -405,8 +405,8 @@
         <!--Other-->
         <netty.version>4.0.30.Final</netty.version>
         <netty.version.range>[4.0.30, 5.0.0)</netty.version.range>
-        <guava.version>20.0</guava.version>
-        <guava.version.range>[20.0,21.0)</guava.version.range>
+        <guava.version>23.0</guava.version>
+        <guava.version.range>[23.0, 24.0)</guava.version.range>
```

Comment:
Is this update compatible with current guava version in SP ?

Reply:
SP 4.1.0 packs 2 Guava versions, v18.0.0 & v20.0.0. As per changes logs (for [v21](https://github.com/google/guava/wiki/Release21), for [v22](https://github.com/google/guava/wiki/Release22), for [v23](https://github.com/google/guava/wiki/Release23)) unless you are using deprecated methods, migrating from v18->v23 or v20->v23 won't break.

Reply:
Since the new APIs used in this PR was introduced in v21, I downgraded the Guava version to v21. See commit efdd747.